libraries/consul_config.rb

```diff
@@ -20,10 +20,13 @@ class ConsulConfig < Chef::Resource
       attribute(:path, kind_of: String, name_attribute: true)
       # @!attribute owner
       # @return [String]
-      attribute(:owner, kind_of: String, default: 'consul')
+      attribute(:owner, kind_of: String, default: lazy { node['consul']['config']['owner'] })
```

Comment:
Any reason why we can't leave this simply as it was and rely on the value being passed through [in the default recipe](https://github.com/johnbellone/consul-cookbook/blob/master/recipes/default.rb#L54)?